src/content/docs/synthetics/synthetic-monitoring/using-monitors/monitor-downtimes-disable-monitoring-during-scheduled-maintenance-times.mdx

```diff
@@ -20,9 +20,15 @@ Schedule **monitor downtimes** to specify times that your synthetic [monitors](/
 * Service interruptions
 
 <Callout variant="important">
-  During synthetic monitor downtimes, your selected monitors stop running until their scheduled end time. To temporarily disable alerts without pausing your monitors, [mute them](/docs/synthetics/new-relic-synthetics/using-monitors/alerting-synthetics#mute) instead.
+  During synthetic monitor downtimes, your selected monitors stop running until their scheduled end time. To temporarily disable alerts without pausing your monitors, [mute them](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications) instead.
 </Callout>
 
+## Monitor downtime tips [#tips]
+
+A monitor downtime has a maximum duration of 24 hours. Monitor downtime windows larger than 24 hours will need to be split across multiple monitor downtime configurations.
+
+If the start and end times of a monitor downtime span midnight, the monitor downtime will last until the next day at the end time specified. For example, a start time of 11:30PM and an end time of 5:00AM would start on the day listed and last until 5:00AM the next day.
```

Comment:
Do we even need to say that downtimes span midnight?

This whole paragraph seems unnecessary...

Reply:
We've received docs feedback that indicates confusion in this part of the configuration. It may be best to include this paragraph, but I'm open to suggestions on how we can improve it.

Reply:
Let's keep it then!